src/hera/workflows/artifact.py

```diff
@@ -24,6 +25,13 @@
 )
 
 
+class LoaderEnum(Enum):
```

Comment:
Small - can we call this `Loader`? This would be consistent with the other use of `Enum` such as `ImagePullPolicy`

Reply:
Just in case we may use `Loader` in other places in the future, an alternative is `ArtifactLoader`